src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -54,5 +65,28 @@ private LogEventLevel ValidateLogLevel(string levelText)
                     throw new ValidationException("The log level options are [Error, Warning, Info, Debug, Trace]");
             }
         }
+
+        private int[] ValidateThresholds(int[] thresholdList) 
+        {
+            foreach(int threshold in thresholdList) 
+            {
+               if(threshold > 100) 
+               {
+                   throw new ValidationException("The threshold can never be > 100");
+               } else if (threshold < 0)
+               {
+                   throw new ValidationException("The threshold can not be < 0");
+               }
+            }
+
+            if (thresholdList[0] >= thresholdList[1] || 
+                thresholdList[1] >= thresholdList[2] || 
+                thresholdList[0] >= thresholdList[2]) 
```

Comment:
When `0 >= 1` and `1 >= 2` , 0 is always >= 2. So the last conditional is not necessary.